src/main/java/com/codeless/api/automation/entity/security/UserDto.java

```diff
@@ -0,0 +1,26 @@
+package com.codeless.api.automation.entity.security;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import javax.persistence.Column;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+@Data
+@Accessors(chain = true)
+@JsonView(View.ALLOWED.class)
+public class UserDto {
+
+  @Column(name = "username")
+  @NotEmpty
+  @Email
+  private String username;
+
+  @Size(max = 100, min = 5, message = "Invalid password size. Min - 5, Max - 100.")
+  @NotEmpty
+  @Column(name = "password")
```

Comment:
1. remove `@Column` in DTO.
2. remove Dto from name
3. move dto page in backend

Reply:
renamed to AppUser and moved to Dto